src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int ballNumberBound = 100;

```

Comment:
[Follow naming convention for constants](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names)